jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/util/KubernetesResourceFragments.java

```diff
@@ -160,27 +151,33 @@ private static Map<String, Object> readAndEnrichFragment(
 
     // Kind
     boolean nameFromFileIsKind = false;
+    final MappingConfig mcFromType = typeFromFile == null ? null : FILENAME_TO_MAPPING_CONFIG_MAPPER.get(typeFromFile.toLowerCase());
```

Comment:
Minor: Maybe find a better name? as Mapping...mapper sounds confusing to me

Reply:
what part are you referring to?
FILENAME_TO_MAPPING_CONFIG_MAPPER?
Maybe the case makes it confusing, this is the name of a `Map` that converts filenames (`String`) to `MappingConfig`s.
We could strip out the `_MAPPER` part, but IMHO that might make it more confusing